util/api/api.js

```diff
@@ -2,7 +2,7 @@ import axios from '~/plugins/axios';
 import { IS_TESTNET, EXTERNAL_HOSTNAME } from '@/constant';
 
 const LIKECOIN_API_BASE = IS_TESTNET ? 'https://api.rinkeby.like.co' : 'https://api.like.co';
-const LIKECOIN_MISC_API_BASE = `https://${EXTERNAL_HOSTNAME}`;
+const LIKECOIN_MISC_API_BASE = IS_TESTNET ? 'https://rinkeby.like.co' : `https://${EXTERNAL_HOSTNAME}`;
```

Comment:
We should check `TEST_MODE` instead of `IS_TESTNET` since this issue only happen in local dev